docs/source/array-random.rst

```diff
@@ -0,0 +1,75 @@
+Random Number Generation
+========================
+
+Dask's random number routines produce pseudo random numbers using a backend
+library via combinations of a ``BitGenerator`` to create sequences and a 
+``Generator`` to use those sequences to sample from different statistical
+distributions:
```

Comment:
This feels slightly confusing, because it's not clear how the "backend" library comes into play. Maybe leave that point out and clarify that point below? E.g:
 
"Dask's random number routines produce pseudo random numbers using combinations of a ``BitGenerator`` to create sequences and a ``Generator`` to use those sequences to sample from different statistical distributions:"

Or

"Dask produces pseudo-random numbers using a ``BitGenerator`` object to create random bit sequences and a ``Generator`` object to transform those bits into random values from a statistical distribution."

Reply:
Aye. It was a confusing first sentence. Went with the first version. Thank you